package.json

```diff
@@ -8,7 +8,7 @@
     "git": "git@git.github.com:joyent/sdc-adminui.git"
   },
   "engines": {
```

Comment:
Trent Mick
Oct 19 11:35 PM

"engines": {
    "node": ">=0.10"
  },
Unless you know of other parts of this that will break compat with node 0.10.